### DISCUSSION TITLE
Community Node Support

### DISCUSSION THREAD

**Discussion by badi95** [guydavis/machinaris#746] [Q&A]
Spacefarmers.io recently came out with a community node that you can point to, instead of hosting your own node. Is it possible to configure machinaris to work with the community node? 
https://spacefarmers.io/wiki/poolinfo/community_node

**Comment by guydavis** [comment]
Sorry, currently Machinaris depends heavily upon fullnode services including the wallet for charting/showing balances and listing transactions.  As such, custom Chia farmers such as Spacefarmer and Flexfarmer are not supported by Machinaris.
